### PR TITLE
feat: Add 2020 roadmap

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <gaoce@caicloud.io>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -1,43 +1,14 @@
...
-	* Ideally users should be able to export and reuse trained models from a common storage
-
-# Test and Release Infrastructure
-
-* Improve e2e test coverage
-* Improve test harness
-* Enhance release process; adding automation (see https://bit.ly/2F7o4gM) 
+- Delete Suggestion deployment after Experiment is finished [#1061](https://github.com/kubeflow/katib/issues/1061)
+- Save Suggestion state after deployment is deleted [#1062](https://github.com/kubeflow/katib/issues/1062)
+- Reconsider the design of Trial Template [#906](https://github.com/kubeflow/katib/issues/906)
```
Comment (andreyvelich) [inline ROADMAP.md:14]: Maybe also add: Change design of NAS Model Constructor? 

As you know, we have model constructor inside Training Container. I think it is not very useful for the user.

If we can construct model somewhere else (Suggestion or Init Container) and send, for example, json representation of the model directly to the training container, user doesn't need to include this part to the training container.

I will create an issue about it soon.

**Comment by andreyvelich** [comment]
Also, I found that we have something in TODO for the current NAS RL suggestion: https://github.com/kubeflow/katib/tree/master/pkg/suggestion/v1alpha3/NAS_Reinforcement_Learning#to-do.

What do you think, should we add something to the ROADMAP?
For example, step 1: "Add 'micro' mode, which means searching for a neural cell instead of the whole neural network. "

/cc @johnugeorge @gaocegege

File: ROADMAP.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gaocegege) [reply ROADMAP.md:14]: SGTM, I will open an issue and add it.

[the remark this answers is not in the document]
Comment (gaocegege) [reply ROADMAP.md:14]: SGTM

**Comment by gaocegege** [comment]
@andreyvelich Do we have the bandwidth for the TODO in ENAS? I think we can add it but in low priority. WDYT

File: ROADMAP.md (hunk 1 above)
Comment (gaocegege) [reply ROADMAP.md:14]: LGTM. Looking forward to your issue.

**Comment by johnugeorge** [comment]
Fixes: #1104

**Comment by andreyvelich** [comment]
@gaocegege Thank you for the changes!.
Regarding to ENAS, yes I have few thoughts that can be improved.
Should I add them to the ROADMAP or issue is enough?

**Pull request merged**